src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/AttachExceptionRecordToExistingCaseTest.java

```diff
@@ -193,6 +193,39 @@ public void should_attach_exception_record_to_case_by_attach_to_case_reference()
         verifyExceptionRecordAttachesToCase(null);
     }
 
+    @Test
+    public void should_attach_exception_record_to_case_by_searchCaseReference_and_attachToCaseReference()
+        throws Exception {
+        //given
+        // set searchCaseReference as targetCase id
+        CaseDetails targetCase = ccdCaseCreator.createCase(emptyList(), Instant.now());
+
+        // set attachToCaseReference value with attachToCaseDetails id, which should be ignored
+        CaseDetails attachToCaseDetails = ccdCaseCreator.createCase(emptyList(), Instant.now());
```

Comment:
not sure comments and code are in sync. Where are we setting the `searchCaseReference` value

Reply:
I'll update the test